extensions/amp-onetap-google/validator-amp-onetap-google.protoascii

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-onetap
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-onetap"
+    version: "0.1"
+    version: "latest"
+    requires_usage: EXEMPTED
+    deprecated_allow_duplicates: true
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-onetap>
+  html_format: AMP
+  tag_name: "AMP-ONETAP"
+  requires_extension: "amp-onetap"
+  # data-* is generally allowed, but it's not generally mandatory.
+  attrs: {
+    name: "data-href"
+    mandatory: true
+  }
+  attr_lists: "extended-amp-global"
+  amp_layout: {
```

Comment:
How is layout applied in the different boxes? As far as I understand, this should always be `fixed` or `fixed_height` since it floats?

Reply:
only support nodisplay now.